packages/framework/src/Model/Seo/HreflangLinksFacade.php

```diff
@@ -123,6 +125,31 @@ public function getForBrand(Brand $brand, int $currentDomainId): array
         return $result;
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Blog\Article\BlogArticle $blogArticle
+     * @param int $currentDomainId
+     * @return \Shopsys\FrameworkBundle\Model\Seo\HreflangLink[]
+     */
+    public function getForBlogArticle(BlogArticle $blogArticle, int $currentDomainId): array
```

Comment:
there is a `BlogArticleDomain::$visible` attribute - should not it be taken into account?